pypesto/sampling/geweke_test.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+from scipy.stats import norm
+
+
+def spectrum(x: np.array,
```

Comment:
can you please also for the other algorithms give sources? thx!

Reply:
They are all matlab functions in https://github.com/mjlaine/mcmcstat